static/app/views/organizationGroupDetails/eventToolbar.tsx

```diff
@@ -116,7 +117,7 @@ class GroupEventToolbar extends Component<Props> {
       Math.abs(+moment(evt.dateReceived) - +moment(evt.dateCreated)) > latencyThreshold;
 
     return (
-      <Wrapper>
+      <StyledDataSection>
```

Comment:
Getting rid of this custom Wrapper in favor of DataSection. Every part of this page should use that component instead to keep the margins/padding consistent.